backend/src/resource/notificacao-resource.js

```diff
@@ -25,13 +25,17 @@ const buscarPessoaDadosBasicos = async (nome, nomeDaMae) => models.Pessoa.findAl
 });
 
 const consolidarSuspeito = async (suspeito) => {
-  const {
+  var {
```

Comment:
Para limitar o contexto, seria melhor utilizar let
```suggestion
  let {
```

Reply:
Alterado.